### ISSUE TITLE
default demo page

### ISSUE THREAD

**Issue by Haroenv** [bullgit/bullgit.github.io#56]
We should make a default demo page, like the bootstrap one we use currently for some projects (https://bullg.it/breath.js/ for example), and replace all of those by a default cool page.

**Comment by LukyVj** [comment]
Agree, but I think we should have a kind of bullgit styleguide first

**Comment by JohJakob** [comment]
@LukyVj @Haroenv I would love to write a style guide :smile:
